src/components/images.js

```diff
@@ -0,0 +1,36 @@
+//what use effect and I using here
+import React, { useState } from "react";
+import "./Breeds.css";
+
+/*fetch data method 2 */
+export default function Breeds() {
+  // image data
+  const [url, setUrl] = useState("");
+  console.log(data);
+
+  function fetchData2() {
```

Comment:
We should aim to use more descriptive function names, even if they're slightly more verbose. What does the 2 refer to here? It would be best practice to be specific for whoever reads the code next (and always remember that person is very likely to be your future self!)